operator-common/src/main/java/io/strimzi/operator/common/model/Ca.java

```diff
@@ -89,10 +89,21 @@ public enum SecretEntry {
         }
 
         /**
-         * @return The suffix of the entry name in the Secret
+         *
+         * @param prefix to use for the certificate Secret key
+         * @return a certificate Secret key with the correct suffix
```

Comment:
I thinks this method deserves more explanation. It's returning the key by concatenating prefix + suffix it's not just about returning with "correct suffix", or?

Reply:
Here by "correct suffix" I mean the correct suffix for the type of data we are storing, e.g. cert vs key vs p12 etc.

What about this:
```suggestion
         /**
         * Build the Kubernetes Secret key to use with this type of SecretEntry.
         *
         * @param prefix to use for the certificate Secret key
         * @return a certificate Secret key with the provided prefix and the suffix of this type of SecretEntry
```

Reply:
Yes ... I think it's clearer!

Reply:
I've updated it